apps/desktop/src/database/artifacts/runtime/seed.sql

```
@@ -142,3 +142,9 @@ INSERT INTO built_in_tools (
 )
 SELECT 'visualize_read_me', 'VisualizeReadMe', '读取 ShowWidget 生成规范', 1, 'low', NULL
 WHERE NOT EXISTS (SELECT 1 FROM built_in_tools WHERE tool_id = 'visualize_read_me');
+
+INSERT INTO built_in_tools (
+    tool_id, display_name, description, enabled, risk_level, config_json
+)
+SELECT 'ask_user_question', 'AskUserQuestion', '向用户提出结构化问题', 1, 'low', NULL
+WHERE NOT EXISTS (SELECT 1 FROM built_in_tools WHERE tool_id = 'ask_user_question');
```

SQLFluff (4.2.1)

[error] The 'WHERE' keyword should always start a new line.

(LT14)